tests/std/tests/Dev11_0836436_get_time/test.cpp

```diff
@@ -944,3 +909,68 @@ void test_gh_4820() {
         assert(wss.str() == L"\x043a%\x043e%\x0448%E\x043a%O\x0430");
     }
 }
+
+void test_gh_4882() {
+    // GH-4882 <iomanip>: std::put_time should set failbit on invalid/out-of-range tm struct, instead of crash

```

Comment:
To avoid confusion, the title of this issue should be updated, and then this comment should mirror it. It would be fine to just have the issue say that we shouldn't crash for out-of-range values, no need to say what we should do.